dub.sdl

```diff
@@ -0,0 +1,10 @@
+name "dustmite"
+authors "Vladimir Panteleev"
+license "Boost"
+targetPath "."
+sourceFiles "dustmite.d" "polyhash.d" "splitter.d"
+
+configuration "application" {
+	targetType "executable"
+	mainSourceFile "dustmite.d"
+}
```

Comment:
What is the purpose of a `configuration` block if `application` is the only meaningful way to build this project?

Reply:
I think this isn't resolved, unless there is an obvious answer that I'm missing.

Reply:
Ahh, sorry. Hang on.

Reply:
Done.